app.js

```diff
@@ -53,11 +68,55 @@ var attributionOsm = '<a href="https://www.openstreetmap.org/copyright">OpenStre
   {id: 'Image_2009-2012', title: 'Orthofoto Tirol: 2009–2012'},
   {id: 'Image_2013-2015_RGB', title: 'Orthofoto Tirol: 2013–2015'},
   {id: 'Image_2013-2015_CIR', title: 'Orthofoto Tirol: 2013–2015 <abbr title="photographisches Infrarot">CIR</abbr>'}
-].map(function(options) {
+].forEach(function(options) {
   var layer = L.tileLayer.wms('https://gis.tirol.gv.at/arcgis/services/Service_Public/orthofoto/MapServer/WMSServer', {
     layers: options.id,
     format: 'image/jpeg',
     attribution: attribution
   });
   layers.addBaseLayer(layer, options.title);
+  allMapLayers[options.id] = layer;
 });
+
+[
+  {id: 'P_BZ_BASEMAP_TOPO', title: 'South Tyrol Base Map'},
```

Comment:
I would have used German layer names, but I'm happy to take the English ones as well. :smile: